cidc_api/models.py

```diff
@@ -455,6 +455,14 @@ def trial(cls):
     # The object URI for the raw excel form associated with this upload
     gcs_xlsx_uri = Column(String, nullable=False)
 
+    def alert_upload_success(self):
+        """Send an email notification that an upload has succeeded."""
+        # (import these here to avoid a circular import error)
+        import emails, gcloud_client
+
+        # Send admin notification email
+        gcloud_client.send_email(**emails.new_upload_alert(self))
```

Comment:
also, is there a reason not to import and use `gcloud_client` in the `email` module directly?

Reply:
Nope! I can refactor.